lib/components/base/DropdownSelect.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <div class="dropdown-overlay" @click="hide" />
+  <div
+    class="custom-dropdown-select"
+    :class="{
+      'round-bottom': !dropdownOpen,
+      'flat-bottom': dropdownOpen,
```

Comment:
does this need to be 2 seperate classes? why not just have the round bottom be the default (so the class can be removed) and just set flat bottom when it it is open